echopype/convert/ek80.py

```diff
@@ -545,6 +546,31 @@ def _export_zarr(self, save_settings, file_idx=0):
         else:
             self._set_groups(raw_file, out_file, save_settings=save_settings)
 
+    def _copy_group(self, src_file, trg_file):
```

Comment:
Could you give this method a better name so that it is more descriptive of what the function is? It is hard coded to only copy the Vendor group so the name does not make sense.